db/migrate/20180829130513_update_kexec_uri.rb

```diff
@@ -0,0 +1,18 @@
+class UpdateKexecUri < ActiveRecord::Migration[5.0]
+  def up
+		ProvisioningTemplate.without_auditing do
+      ["Discovery Red Hat kexec", "Discovery Debian kexec"].each do |template_name|
+        if t = ProvisioningTemplate.unscoped.find_by_name(template_name)
```

Comment:
I would probably go the way https://github.com/inecas/foreman/blob/5963f341d505d73d96b616db02af3d93e06b74c5/db/migrate/20170629080604_change_preseed_default_pxe_grub2_kind.rb#L9 does it, so that we still get audit logs for existing installations: might be useful for the seeing this changes.

Reply:
Ok done thanks.